code/components/gta-net-five/src/MumbleVoice.cpp

```diff
@@ -705,6 +705,29 @@ static HookFunction hookFunction([]()
 			}
 		});
 
+        fx::ScriptEngine::RegisterNativeHandler("MUMBLE_REMOVE_VOICE_TARGET_CHANNEL", [](fx::ScriptContext& context)
+        {
+            auto id = context.GetArgument<int>(0);
+            auto channel = context.GetArgument<int>(1);
+
+            if (id >= 0 && id < 31)
+            {
+                auto targetChannel = fmt::sprintf("Game Channel %d", channel);
+                std::vector<VoiceTargetConfig::Target>& targets = vtConfigs[id].targets;
```

Comment:
Does vtConfigs get accessed by any other thread here i.e. potentially need locking?

Also, long types sound like they'd be better off with `auto&`.

Reply:
Doesn't look like it, vtConfigs is only ever modified by natives, which is sent to `MumbleClient::UpdateVoiceTarget` locks the client mutex and updates `m_pendingVoiceTargetUpdates`